### PR TITLE
[DEVX-1812] Document visibility configuration

### PR DESCRIPTION
**Pull request by mhan83**
### Description

Describe the `visibility` property for saucectl config

### Motivation and Context

New feature

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by tianfeng92 — APPROVED** [review]

**Review by alexplischke — APPROVED** [review]

File: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md [hunk 1]
```diff
@@ -212,6 +212,27 @@ sauce:
     owner: tunnel_owner_username
 ```
 
+---
+#### `visibility`
```
Comment (alexplischke) [inline docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:216]: Gotta go up a level in hierarchy:
```suggestion
### `visibility`
```

File: docs/web-apps/automated-testing/cypress/yaml/v1.md [hunk 2]
```diff
@@ -236,6 +236,23 @@ sauce:
 ```
 ---
 
+#### `visibility`
```
Comment (alexplischke) [inline docs/web-apps/automated-testing/cypress/yaml/v1.md:239]:
```suggestion
### `visibility`
```

File: docs/web-apps/automated-testing/cypress/yaml/v1alpha.md [hunk 3]
```diff
@@ -234,6 +234,23 @@ sauce:
 ```
 ---
 
+#### `visibility`
```
Comment (alexplischke) [inline docs/web-apps/automated-testing/cypress/yaml/v1alpha.md:237]:
```suggestion
### `visibility`
```

File: docs/web-apps/automated-testing/playwright/yaml.md [hunk 4]
```diff
@@ -223,6 +223,23 @@ sauce:
 ```
 ---
 
+#### `visibility`
```
Comment (alexplischke) [inline docs/web-apps/automated-testing/playwright/yaml.md:226]:
```suggestion
### `visibility`
```

File: docs/web-apps/automated-testing/testcafe/yaml.md [hunk 5]
```diff
@@ -224,6 +224,23 @@ sauce:
 ```
 ---
 
+#### `visibility`
```
Comment (alexplischke) [inline docs/web-apps/automated-testing/testcafe/yaml.md:227]:
```suggestion
### `visibility`
```

**Review by lauramthomas — APPROVED** [review]

**Pull request merged**